org.dupot.beatandmatchtopass.yml

```diff
@@ -0,0 +1,24 @@
+id: org.dupot.beatandmatchtopass
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: beatmatchtopass.x86_64
+finish-args:
+  - --share=ipc
+  - --socket=pulseaudio
+  - --socket=x11
+  - --device=dri
+  - --persist=.local/share/godot/app_userdata/BeatMatchtoPass
+
+modules:
+  - name: BeatMatchtoPass
+    buildsystem: simple
+    build-commands:
+      - install -Dm555 export/linux/beatmatchtopass.x86_64 /app/bin/beatmatchtopass.x86_64
+      - install -Dm644 export/linux/flatpack/512x512.png /app/share/icons/hicolor/512x512/apps/org.dupot.beatmatchtopass.png
+      - install -Dm644 export/linux/flatpack/org.dupot.beatmatchtopass.desktop /app/share/applications/org.dupot.beatmatchtopass.desktop
```

Comment:
https://github.com/imikado/dupotBeatAndMatchToPass/blob/67167b59b4bef2ebe6246400471df0949146bb58/export/linux/flatpack/org.dupot.beatmatchtopass.desktop#L3 should not be absolute path

Reply:
fixed

Reply:
how?

Reply:
still doing it 
![image](https://github.com/flathub/flathub/assets/114441/58dab665-1275-481d-87d2-a170d27b6685)

Reply:
no. that's even worse.

Reply:
i just commit the fix

Reply:
you didn't. you did what I said to NOT do.

Reply:
i delete $FLATHPAK_DEST, only bin/..

Reply:
ok, i remove .x86_64 (for information, it is the default godot generated name )
just commit

Reply:
:facepalm:

Reply:
*hint* `bin` is a directory. it is not part of the executable file name. `Exec` is supposed to be the executable file name, as it is supposed to be located in one of the directories in `PATH`. This was the original request to no have an absolute path.

Reply:
i don't understand: this is the godot executable binary

Reply:
`beatmatchtopass.x86_64` IS.

Reply:
ok
scuse me, but as mentionned, i only copied last published project and change only project name.. i don't know why this was ok for my previous projet but not this one
i commit only executable name